### PR TITLE
Use UBI as base for Dockerfile

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
This patch changes the base image for the Dockerfile to ubi8 to make it
more in-line with OpenShift.

Also removes OC cli install as a Dockerfile step, now that we can
dynamically inject the oc cli as part of multistage tests

Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

**Review by jmencak — COMMENTED** [review]
I don't feel qualified to approve, however, I've built this image successfully.  Not sure I understand why we don't need the openshift-clients now.  Also, I'd recommend consistent spacing/indentation in the Dockerfile before merging.

File: build/Dockerfile
Comment (kpouget) [inline build/Dockerfile:24]: just a style nit, I would cahnge a bit the indentation to better distinguish multiple args (yum/pip3) and multiple commands, something like that:

```suggestion
RUN yum install -y \
        glibc-langpack-en \
        git make jq vim wget \
        python3 python3-devel python3-pip python3-setuptools \
 && pip3 install --no-cache-dir --upgrade setuptools pip \
 && pip3 install --no-cache-dir \
           openshift \
           ansible==2.9.* \
 && yum clean all \
 && rm -rf $HOME/.cache \
 && rm -rf /var/cache/yum
```

**Comment by ArangoGutierrez** [comment]
Test will continue to fail until we can merge https://github.com/openshift/release/pull/12835 
once we can get a `lgtm`here this will fix failing tests for that PR

File: build/Dockerfile
Comment (sjug) [inline build/Dockerfile:18]: Can we move `&&` to the end of the previous line?

**Review by sjug — CHANGES_REQUESTED** [review]
Just one thing IMO

File: build/Dockerfile
Comment (ArangoGutierrez) [reply build/Dockerfile]: done

File: playbooks/gpu_machineset_aws_template.json
Comment (jmencak) [inline playbooks/gpu_machineset_aws_template.json:1]: Agree with the bot, no EOL is a *bad* thing, same below.

Comment (ArangoGutierrez) [reply playbooks/gpu_machineset_aws_template.json]: Ups those files are autogenerated should be there, removed

File: build/Dockerfile
Comment (kpouget) [inline build/Dockerfile:24]: I think the indentation got lost since a previously version?

Comment (ArangoGutierrez) [reply build/Dockerfile:24]: since I moved the `&&` to the line before, now looks like this, but I get what you mean
fixing

Comment (ArangoGutierrez) [reply build/Dockerfile]: Done

**Review by jmencak — CHANGES_REQUESTED** [review]
Can we please have some consistency in the Dockerfile?  Sometimes there are tabs, sometimes spaces.  Sometimes we have '&&' at the end of the line, sometimes at the beginning of the line.  Sometimes we indent by 4 spaces, sometimes by one...  These are all "nits", but I'm afraid they hurt (probably not only my) eyes.

**Comment by ArangoGutierrez** [comment]
> Can we please have some consistency in the Dockerfile? Sometimes there are tabs, sometimes spaces. Sometimes we have '&&' at the end of the line, sometimes at the beginning of the line. Sometimes we indent by 4 spaces, sometimes by one... These are all "nits", but I'm afraid they hurt (probably not only my) eyes.

Agree with you Jiri, only me to blame, I think I have diff settings on VIM on my laptop and PC, that creates some indentation misleading here

**Comment by ArangoGutierrez** [comment]
Changed to all tabs

**Pull request merged**
